CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add `Py::setattr` method. [#2009](https://github.com/PyO3/pyo3/pull/2009)
 - Add `PyCapsule`, exposing the [Capsule API](https://docs.python.org/3/c-api/capsule.html#capsules). [#1980](https://github.com/PyO3/pyo3/pull/1980)
+- Add a mechanism for PyO3 to generate default definition of magic methods. [#2014](https://github.com/PyO3/pyo3/pull/2014)
```

Comment:
TBH this is an internal change rather than something that's meaningful to users. The output is part of `#[pyclass]` for enum, which is what we should have a CHANGELOG entry for. So I would take this one out.

```suggestion
```